src/types.ts

```diff
@@ -49,6 +49,15 @@ export interface MentionLabelProps {
   isByMe: boolean;
 }
 
+// fork note: the minimum amount of information needed to save and restore
+// an *unsent* message as a draft
+export interface DraftMessage {
+  messageId: number | null;
```

Comment:
Can you explain why we need a special DraftMessage type?

Reply:
Sure! `DraftMessage` is used to support save/load of draft messages that have mentions with minimal code changes and maximum compatibility with UIKit. UIKit already has all code related to extracting, displaying, sending, et, of messages with mentions, and it only requires the data present in the `DraftMessage` interface. This is the code that is also used for "editing" messages.

The `DraftMessage` interface is also handy as the interface between Gather-Browser <-> UIKIt, and the draft code.

The alternative would be us implementing additional custom code which handles extracting, saving, and re-loading draft messages that have mentions.